### PR TITLE
ci: update packages

### PR DESCRIPTION
**Pull request by matthiasdiener**
Should resolve spurious CI failures.

### CODE REVIEW

**Review by MTCam — APPROVED** [review]
👍 Thanks Matthias!

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -141,7 +142,7 @@ jobs:
         - uses: actions/checkout@v2
         - name: Install emirge
           run: |
-            [[ $(uname) == Linux ]] && sudo apt-get install -y openmpi-bin libopenmpi-dev
+            [[ $(uname) == Linux ]] && sudo apt-get update && sudo apt-get install -y openmpi-bin libopenmpi-dev
```
Comment (inducer) [inline .github/workflows/ci.yaml:145]: No objection here, though maybe there's a benefit to grabbing this from conda-forge?

Comment (matthiasdiener) [reply .github/workflows/ci.yaml:145]: But this is the emirge case - how would we get emirge to use the conda MPI (from a different env)?

Comment (inducer) [reply .github/workflows/ci.yaml:145]: Fair, I hadn't picked up on that. I was meaning that more for the other CIs, so feel free to disregard this.

**Comment by MTCam** [comment]
Merge: ftw.

**Pull request merged**